README.md

```diff
@@ -75,16 +102,19 @@ Note: `[1:]` is used to strip out the `v` in the version tag, e.g. `v1.1.0` beco
 
 ### TODO
 
-- [ ] Tests :grimacing:
+- [x] Tests :grimacing:
 - [ ] Support Github tokens for private repository lookups
 
 ### Contributing
 
-This repository welcomes contributions. Please fork this repository and file a pull request to the master branch. Your changes must include tests in the `tests/` directory. Run your code through `flake8` and `black` before creating the pull-request.
+This repository welcomes contributions. Please fork this repository and file a pull request to the master branch. Your changes must include any appropriate tests under the `molecule/` directory. Read about how to use Molecule from the official documentation [here][molecule official] and from Jeff Geerling [here][geerlingguy molecule].
```

Comment:
@geerlingguy I found your Ansible + Molecule articles very helpful. Hope it's ok to refer to your article here. And while I have you in this comment, you may also be interested in a GitHub Action I've built to auto-deploy new versions of a collection to Galaxy: https://github.com/marketplace/actions/deploy-ansible-galaxy-collection.